test/plugins/iamauth.js

```diff
@@ -658,4 +607,56 @@ describe('#db IAMAuth Plugin', function() {
       });
     });
   });
+
+  it('returns an error if access token returns non-200 response', function(done) {
```

Comment:
It's a bit confusing that this test has the same name as the earlier test (line 345) `returns an error if access token returns non-200 response`

AFAICT the only difference is the pruned signature of the callback function in this version, perhaps that can just be applied to the previous test unless I'm missing something about the duplication.

Reply:
Good spot. I think this is a dupe. I'll remove it.